concent-builder/repository-prepare.yml

```diff
@@ -0,0 +1,59 @@
+- hosts:
+    - concent-builder
+  vars_files:
+    - consts.yml
+    - repositories.yml
+  tasks:
+    - become:      yes
+      remote_user: builder
+      block:
+        - name:    Create a directory for secrets
+          file:
+            path:  "/home/{{ shared_user }}/build/concent-secrets"
+            state: directory
+
+        - name:    Copy secrets to remote location
+          copy:
+            src:   "{{ local_secret_dir }}/{{ item }}"
+            dest:  "/home/{{ shared_user }}/build/concent-secrets"
+          with_items:
+            - secrets.py
+            - var-secret.yml
+            - devel-concent-golem-network-198b161e7c8a.json
+
+        - name:      Create directories for git repositories
+          file:
+            path:    "{{ data_dir }}/{{ item.key }}"
+            state:   directory
+          with_dict: "{{ repositories }}"
+
+        - name:             Clone concent repositories
+          git:
+            repo:           "{{ item.value.url }}"
+            dest:           "{{ data_dir }}/{{ item.key }}/"
+            clone:          yes
+            update:         yes
+            bare:           yes
+            # FIXME: Do not blindly accept the hostkey
+            accept_hostkey: yes
+          with_dict:        "{{ repositories }}"
+
+        - name:      Remove the build directory to get rid of files from previous builds
+          file:
+            path:    "{{ build_dir }}/*"
+            state:   absent
+
+        - name:      Check out working copies of all repositories for the build
+          file:
+            path:    "{{ build_dir }}/{{ item.key }}"
+            state:   directory
+          with_dict: "{{ repositories }}"
```

Comment:
This step is not necessary (and its description is misleading - we're not checking anything out here, we're just creating directories). `git` task will create these directories automatically if they don't exist.